app/models/taxonomies/organization.rb

```diff
@@ -15,8 +15,9 @@ class Organization < Taxonomy
 
   scope :completer_scope, ->(opts) { my_organizations }
 
-  scope :my_organizations, lambda {
-    conditions = User.current.admin? ? {} : sanitize_sql_for_conditions([" (taxonomies.id in (?))", User.current.organization_and_child_ids])
+  scope :my_organizations, lambda { |user = nil|
```

Comment:
One observation: this stile of links to discussion is not good, as it gets invalid as soon as the diff was outdated. The correct way how to link the things is https://github.com/theforeman/foreman/pull/3732#discussion_r74886627 - you can get it while getting the link to the comment from `Copnversation` tab. This helps a lot co-reviewers following up the discussion. - it even makes nicer links in the text as you see. Thanks